src/Microsoft.AspNet.Server.Kestrel/Http/MessageBody.cs

```diff
@@ -94,7 +94,7 @@ public static bool TryGet(IDictionary<string, StringValues> headers, string name
                 value = values[0];
                 return true;
             }
-            value = String.Join(",", values);
+            value = string.Join(",", values.ToArray());
```

Comment:
Before this change wasn't [this overload of string.Join](https://github.com/dotnet/coreclr/blob/33323a204ab92b5f8762461b8915b7ffa6d89e69/src/mscorlib/src/System/String.cs#L156) being used? Is there a reason to avoid that overload?